tests/test_admin_and_files.py

```diff
@@ -0,0 +1,65 @@
+import os
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+from blog.models import Post, Commentary
+
+
+class AdminSiteBlogTests(TestCase):
+    def setUp(self) -> None:
+        self.admin_user = get_user_model().objects.create_superuser(
+            username="admin.user",
+            password="testdata123456",
+        )
+        self.client.force_login(self.admin_user)
+        self.user = get_user_model().objects.create_user(
+            username="test.user",
+            password="testpassword13579"
+        )
+        self.user_2 = get_user_model().objects.create_user(
+            username="test.user_2",
+            password="testpassword24680"
+        )
+        self.post = Post.objects.create(
+            owner=self.user,
+            title="Test post",
+            content="Some test content"
+        )
+        self.comment = Commentary.objects.create(
+            user=self.user_2,
+            post=self.post,
+            content="Agree"
+        )
```

Comment:
For what purpose you create all these test data, if you do not use it later?

Reply:
that was `Ctrl + C` and `Ctrl + V`, I guess

Reply:
No, just I decided to remove some tests and forgot to delete this data